### ISSUE TITLE
[DOC]: Clarify the benchmarks page

### ISSUE THREAD

**Issue by AGenchev** [serengil/deepface#1275]
### Suggested Changes

I can't suggest exact text, because I couldn't understand the page so to be able to say what is what. But there is something which can be done to improve the usability of this page. 
For example, there is a matrix with title "Performance Matrix **for euclidean_l2** while **alignment is False**". Let's take the "**ArcFace**" column. 
1. I don't get what "**for euclidean_l2**"  in the title does mean - did you perform training of the backbone of the ArcFace using euclidean_l2 distance instead of ArcCosine distance in the loss function then evaluating on LFW using again euclidean_l2 instead of cosine or if not, what did you do ?
2. The first, noname column contains things like "retinaface", "centerface", etc. which look like alignment methods. So what **alignment is False** statement means - that you did not perform alignment using the methods in the noname coumn ? Then these metods were used to perform what ? Only face detection ?
3. Where **alignment is True**, did you train each model with the given alignment applied to the training dataset and on the testing (LFW) dataset or you took a pretrained on whatever dataset model and evaluated it only on aligned (with the given alignment model) LFW dataset ?

**Comment by serengil** [comment]
recommend to read its paper - https://doi.org/10.17671/gazibtd.1399077

everything is explained in there
